aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu

```diff
@@ -263,5 +265,48 @@ TORCH_IMPL_FUNC(_convert_indices_from_csr_to_coo_structured_cuda) (
   }
 }
 
+template <typename scalar_t, typename value_t>
+void _apply_sparse_csr_lu_solve(
+  const at::sparse_csr::SparseCsrTensor& input,
+  const Tensor& other,
+  Tensor& result,
+  int &_singularity) {
+  auto values = input.values();
+  const scalar_t *values_data_ptr = values.data_ptr<scalar_t>();
+  auto crow_indices = input.crow_indices().to(kInt);
+  const int *crow_indices_data_ptr = crow_indices.data_ptr<int>();
+  auto col_indices = input.col_indices().to(kInt);
+  const int *col_indices_data_ptr = col_indices.data_ptr<int>();
+  auto handle = at::cuda::getCurrentCUDASolverSpHandle(); 
+  auto descrA = at::cuda::sparse::CuSparseMatDescriptor();
+
+  const scalar_t *b = other.data_ptr<scalar_t>();
+  int n = crow_indices.numel() - 1; 
+  int nnzA = input._nnz();
+  value_t tol = 0.0;
+  // default reordering of symrcm
+  // Should reorder be an argument provided for users to choose between the following?
+  // symrcm, symamd, csrmetisnd (1, 2, 3)
+  int reorder = 1;
```

Comment:
Should this be 0 (no reordering)? I need to search more about what it does. The documentation suggests that using recording does significant optimization.

Reply:
Let's use `reorder = 0` as it's not clear how exactly it affects the performance.